v3/src/v2/codap-v2-types.ts

```diff
@@ -584,9 +570,6 @@ export function isV2MapLegacyStorage(obj: unknown): obj is ICodapV2MapCurrentSto
 export interface ICodapV2MapLayerBaseStorage {
   _links_: {
     context: IGuidLink<"DG.DataContextRecord">
-    // TODO_V2_IMPORT hiddenCases are not imported
-    // this array was passed right into MST where it is typed as a string array
-    // There are 296 instances where this is a non-empty array in cfm-shared
```

Comment:
I mentioned this elsewhere, but `tHiddenCases` (comment below) is _not_ always empty and so I think it can be imported and then the comment updated.